### PR TITLE
MAINT: Update copyright notice to 2021 (Part II)

### PR DESCRIPTION
**Pull request by Krzmbrzl**
Apparently the first commit (59ae429972c16c377135bcccfee646b7df446933)
did not include all files.

### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

File: plugins/amongus/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Mumble Developers. All rights reserved.
```
Comment (Krzmbrzl) [inline plugins/amongus/CMakeLists.txt:1]: Okay something is clearly not correct here.
The script must be bugged...

File: plugins/amongus/CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+# Copyright 2009-2021 The Mumble Developers. All rights reserved.
```
Comment (Krzmbrzl) [inline plugins/amongus/CMakeLists.txt:1]: Something is not correct here. The script must be bugged
(This file did definitely not exist back in 2009)

File: .ci/azure-pipelines/build_linux.bash [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -ex
 #
-# Copyright 2019-2021 The Mumble Developers. All rights reserved.
+# Copyright 2020-2021 The Mumble Developers. All rights reserved.
```
Comment (Kissaki) [inline .ci/azure-pipelines/build_linux.bash:3]: This changes the source year rather than only updating new files?

**Review by Kissaki — COMMENTED** [review]
What are “funny results”? :D

File: .ci/azure-pipelines/build_linux.bash (hunk 3 above)
Comment (Krzmbrzl) [reply .ci/azure-pipelines/build_linux.bash:3]: Yes - that's what the old version of the script messed up and the new one fixes

**Comment by Krzmbrzl** [comment]
@Kissaki "funny results" are for instances ones as I commented on in my review comment in this PR

**Pull request merged**